src/utilities/math.ts

```diff
@@ -76,37 +76,81 @@ export function getPercentiles(histogram: CARTA.IHistogram, ranks: number[]): nu
     return calculatedPercentiles;
 }
 
-export function scaleValue(x: number, scaling: FrameScaling, alpha: number = 1000, gamma: number = 1.5) {
+function errorFunction(x: number, c: number, x0: number) {
+    const y = Math.exp(c * (x - x0));
+    return y / (y + 1);
+}
+
+export function scaleValue(x: number, scaling: FrameScaling, alpha: number = 1000, gamma: number = 1.5, bias: number = 0, contrast: number = 1,
+    biasContrastMode: boolean = false, smoothedBias: number = 0, smoothedContrast: number = 0) {
```

Comment:
can we not just use the same arguments (`bias` and `contrast` for both modes, but use `biasContrastMode` to determine which function to use?

Reply:
Combined the parameters.